src/modules/holder/pages/Home/Home.tsx

```diff
@@ -41,21 +40,49 @@ export const Home: FC = () => {
       </>
     )
   }
+
+  const validTickets = data.filter((credentialItem) => {
+    const credentialSubject = (credentialItem as StoredW3CCredential)?.credentialSubject
+    if (Date.parse(credentialSubject?.eventDate) > Date.now()) {
+      return credentialItem
+    }
+  })
+
+  const expiredTickets = data.filter((credentialItem) => {
+    const credentialSubject = (credentialItem as StoredW3CCredential)?.credentialSubject
+    if (Date.parse(credentialSubject?.eventDate) <= Date.now()) {
+      return credentialItem
+    }
+  })
+
+  const getCredential = (credentialSubject: any, credentialItem: any) => {
+    const credential: Credential = {
+      title: credentialSubject?.eventName,
+      date: credentialSubject?.eventDate,
+      time: credentialSubject?.eventTime,
+      credentialId: (credentialItem as StoredW3CCredential)?.id,
+    }
+
+    return credential
+  }
+
+  const getTicketCards = (array: any, isActive: boolean) =>
+    array.map((credentialItem: any, index: any) => {
+      const credentialSubject = (credentialItem as StoredW3CCredential)?.credentialSubject
+
+      const credential = getCredential(credentialSubject, credentialItem)
+      return <TicketCard key={index} credential={credential} isActive={isActive} />
+    })
+
   return (
     <>
       <Header title="Your tickets" />
 
-      <Container isGrid>
-        {data &&
-          data.map((credentialItem, index) => {
-            const credential: Credential = {
-              title: getTitles((credentialItem as StoredW3CCredential)?.type),
-              date: (credentialItem as StoredW3CCredential)?.issuanceDate || '',
-              credentialId: (credentialItem as StoredW3CCredential)?.id,
-            }
-            return <Card key={index} credential={credential} />
-          })}
-      </Container>
+      <Container isGrid>{validTickets && getTicketCards(validTickets, true)}</Container>
+
+      {expiredTickets.length !== 0 && <Header title="Expired tickets" />}
+
+      <Container isGrid>{expiredTickets && getTicketCards(expiredTickets, false)}</Container>
```

Comment:
use object if there is more than one args because from function call it is hard to understand what means `true/false`